<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PRIMEIRA MATRIZ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaMatA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaMatB">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SEGUNDA MATRIZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaResultado">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarMatA">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>101</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatA">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>101</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuantidadeColunasMatA">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>71</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>71</y>
      <width>112</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Colunas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Linhas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuantidadeLinhasMatA">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>41</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuantidadeLinhasMatB">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>41</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>41</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Linhas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatB">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>101</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApagarMatB">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>101</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuantidadeColunasMatB">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>71</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>71</y>
      <width>112</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Colunas</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>textEditSaidaMatA</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>textEditSaidaMatB</zorder>
   <zorder>label_5</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>textEditSaidaResultado</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
